### DISCUSSION TITLE
pdf_screaper_graph_OPENAI with problem

### DISCUSSION THREAD

**Discussion by probe365** [ScrapeGraphAI/Scrapegraph-ai#510] [General]
Friends,
First of all, I'm happy to learn about ScrapeGraph platform, but I have had a problem with the results. I have had an error message as following: "UnboundLocalError: cannot access local variable 'chunk' where it is not associated with a value". Could you, please, help me in this case?
Many thanks in advance,
Paulo
proberto@sibra.com.br

**Comment by VinciGit00** [comment]
please update to the new version

**Reply by probe365** [reply]
OK. Thanks!!!
Paulo
